File: epi_judge_cpp/regular_expression.cc
#include <string>

#include "test_framework/generic_test.h"
using std::string;

bool IsMatch(const string &regex, const string &s) {
  // TODO - you fill in here.
  return true;
}

int main(int argc, char *argv[]) {
  std::vector<std::string> args{argv + 1, argv + argc};
  std::vector<std::string> param_names{"regex", "s"};
  return GenericTestMain(args, "regular_expression.cc",
                         "regular_expression.tsv", &IsMatch,
                         DefaultComparator{}, param_names);
}

File: epi_judge_python/regular_expression.py
from test_framework import generic_test


def is_match(regex: str, s: str) -> bool:
    # TODO - you fill in here.
    return True


if __name__ == '__main__':
    exit(
        generic_test.generic_test_main('regular_expression.py',
                                       'regular_expression.tsv', is_match))
